PULL_REQUESTS.md

```diff
@@ -0,0 +1,203 @@
+    -   [Table of Contents](#table-of-contents)
+    -   [Pull Request Creation](#pull-request-creation)
+        -   [Branch Naming](#branch-naming)
+        -   [Changeset Requirements](#changeset-requirements)
+        -   [Conventional Commits](#conventional-commits)
+    -   [Pull Request Template](#pull-request-template)
+    -   [Labels and Their Meanings](#labels-and-their-meanings)
+    -   [Pull Request Review Process](#pull-request-review-process)
+        -   [Review Timing](#review-timing)
+        -   [Review Expectations](#review-expectations)
+        -   [Review Etiquette](#review-etiquette)
+            -   [For Reviewers](#for-reviewers)
+            -   [For PR Authors](#for-pr-authors)
+            -   [Resolving Disagreements](#resolving-disagreements)
+    -   [Merge Criteria](#merge-criteria)
+    -   [Specific Requirements by Element Type](#specific-requirements-by-element-type)
+        -   [New Components](#new-components)
+            -   [Documentation](#documentation)
+            -   [API Documentation](#api-documentation)
+            -   [Technical Requirements](#technical-requirements)
+
+## Pull Request Creation
+
+### Branch Naming
+
+We use a straightforward branch naming convention:
+
+-   `[username]/[short-description]` (e.g., `alex/fix-dropdown-bug`)
+-   If referencing a known issue, incorporate the issue number (e.g., `alex/123-fix-dropdown-bug`)
+
+### Changeset Requirements
+
+For PRs that add or update a component:
+
+-   Must include a changeset to trigger the release train and update the CHANGELOG
+-   Changeset type should be one of:
+    -   `patch` - for bug fixes only
+    -   `minor` - for new components or new APIs in an existing component
+    -   `major` - for breaking changes to a component or public library API
+
+### Conventional Commits
+
+We follow the Conventional Commits standard to make change tracking predictable and automate versioning:
+
+Format: `type(component?): subject`
+
+The component is optional but should reference the package you are updating.
+
+Types include:
+
+-   `feat`: New features or enhancements
+-   `fix`: Bug fixes
+-   `docs`: Documentation changes
+-   `style`: Formatting, linting (not CSS changes)
+-   `chore`: Build tooling, repo management, dependency updates
+-   `perf`: Performance improvements
+-   `test`: Adding or updating tests
+
+Examples:
+
+-   `feat(sp-card): add shadow styles for theme consistency`
+-   `fix(sp-action-menu): correct arrow key navigation in nested menus`
+-   `docs: clarify how to submit bug reports`
+
+For breaking changes, add a `!` after the type/scope:
+
+-   `feat(sp-button)!: change API for icon placement`
+
+## Pull Request Template
+
+When creating a pull request, you'll be presented with our template. Complete all sections to the best of your ability, including:
+
+-   Description of the changes
+-   Related issues (using proper GitHub keywords to auto-close issues)
```

Comment:
@najikahalsema do you want more here? Also i physically added `fixes` to the PR template itself
```suggestion
-   Related issues (using proper GitHub keywords to auto-close issues i.e. `fixes`, `resolves`, or `closes`)
```

Reply:
I think that should be good. Maybe we add a link to the docs on "github keywords"? here's the link: 
![image](https://github.com/user-attachments/assets/5bf3d552-9b6d-4780-a156-25ad3ea988df)